app/src/main/java/de/westnordost/streetcomplete/data/quest/OsmTaggings.kt

```diff
@@ -0,0 +1,19 @@
+package de.westnordost.streetcomplete.data.quest
```

Comment:
Could you put this into `de.westnordost.streetcomplete.data.meta.OsmTaggings`?

Reply:
Of course!